admin/requirements.txt

```diff
@@ -3,9 +3,10 @@ c2cwsgiutils
 deform
 plaster_pastedeploy
 pyproj
-pyramid == 1.9.1
-pyramid_debugtoolbar
-pyramid_jinja2
-pyramid_retry
+pyramid
+pyramid-debugtoolbar
+pyramid-jinja2
+pyramid-retry
```

Comment:
why replacing '-' with '_', I'm curious

Reply:
pip do some mixage and rename when the names isn't the same everywhere, I change back to have the original names everywhere: 
```
pyramid_debugtoolbar
pyramid_jinja2
pyramid-retry
```